### PR TITLE
WIP: Adds laravel 5.6 support

### PR DESCRIPTION
**Pull request by sixlive**
Waiting on https://github.com/orchestral/testbench to release 3.6 for final testing

### CODE REVIEW

**Comment by ToxicToast** [comment]
When will the PR getting merged into master?

**Comment by sixlive** [comment]
@toxictoast as soon as [orchestral/testbench](https://github.com/orchestral/testbench) is tagged for 3.6 and we can verify that tests pass. 

Looks like sometime today or tomorrow.

**Comment by fgreinus** [comment]
What a chain of update-dependencies :laughing:

File: composer.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     ],
     "require": {
         "php": "^7.0",
-        "illuminate/support": "5.5.*"
+        "illuminate/support": "5.5.*|5.6.*"
     },
     "require-dev": {
-        "orchestra/testbench": "3.5.*",
```
Comment (freekmurze) [inline composer.json:27]: Shouldn't we use `3.5.*|3.6.*` here so the tests can use Laravel 5.5 when using lower deps?

Comment (sixlive) [reply composer.json:27]: Good catch!

**Comment by guillaumebriday** [comment]
It's all green ! Thanks !

Migration to 5.6 was so easy from 5.5 👍

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**